docker/compose-controller-spark-sql-single.yaml

```diff
@@ -62,6 +62,8 @@ services:
       - ${DWH_ROOT}:/dwh
     environment:
       - JAVA_OPTS=$JAVA_OPTS
+      # This is to turn this on in e2e but leave it off in the default config.
+      - FHIRDATA_SINKFHIRSERVERURL=$FHIRDATA_SINKFHIRSERVERURL
```

Comment:
I am assuming this will be automatically picked up by the spring configuration `sinkFhirServerUrl`.